src/objects/EscalationComment__c.object

```diff
@@ -70,6 +70,74 @@
+        <label>Created By Contact</label>
+        <referenceTo>Contact</referenceTo>
+        <relationshipLabel>Escalation Comments</relationshipLabel>
+        <relationshipName>Escalation_Comments</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Created_By_Link__c</fullName>
+        <description>Link to the user or contact that created the comment</description>
+        <externalId>false</externalId>
+        <formula>IF(NOT(ISBLANK(Created_By_Contact__c)), HYPERLINK(&apos;/&apos; + Created_By_Contact__c, Created_By_Contact__r.FirstName + &apos; &apos; + Created_By_Contact__r.LastName, &apos;_self&apos;),
+	(IF(NOT(ISBLANK(Created_By_User__c)), HYPERLINK(&apos;/&apos; + Created_By_User__c, Created_By_User__r.FirstName + &apos; &apos; +  Created_By_User__r.LastName, &apos;_self&apos;),
+	IF(NOT(ISBLANK(CreatedById)), HYPERLINK(&apos;/&apos; + CreatedById, CreatedBy.FirstName + &apos; &apos; +  CreatedBy.LastName, &apos;_self&apos;), &apos;&apos;)
+	)))</formula>
+        <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs>
+        <inlineHelpText>Link to the user or contact that created the comment</inlineHelpText>
+        <label>Created By</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Created_By_Text__c</fullName>
+        <description>The text about who created the comment</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The text about who created the comment</inlineHelpText>
+        <label>Created By Text</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Created_By_User__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The user that created the comment. Null if created by a contact</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The user that created the comment. Null if created by a contact</inlineHelpText>
+        <label>Created By User</label>
+        <referenceTo>User</referenceTo>
+        <relationshipName>Escalation_Comments</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Created_By__c</fullName>
+        <externalId>false</externalId>
+        <formula>BLANKVALUE(Created_By_Contact__r.FirstName + &apos; &apos; + Created_By_Contact__r.LastName,
+	BLANKVALUE(Created_By_User__r.FirstName + &apos; &apos; + Created_By_User__r.LastName,
+	$User.FirstName + &apos; &apos; +  $User.LastName))</formula>
```

Comment:
I don't think we want to use `$User` here because that may not be who actually created the comment.  Instead use `CreatedBy.FirstName` and `CreatedBy.LastName` to get the user that actually created the comment.